File: OrtalamaBulma.py
def sum_(x):
    toplam = 0

    for i in x:
        toplam +=i
    return (toplam,len(x))

def average(x:tuple):
    toplam = x[0]
    eleman_sayisi = x[1]

    ortalama = toplam/eleman_sayisi

    return ortalama

ilk = int(input("Başlama değeri : "))
son = int(input("Son değer : "))

liste = [i for i in range(ilk,son+1)]

print(average(sum_(liste)))

File: OrtalamaBulma.cpp
#include <iostream>

using namespace std;

int main() {
    int ilk, son;

    cout << "Başlama değeri: ";
    cin >> ilk;

    cout << "Son değer: ";
    cin >> son;

    int toplam = 0;
    int elemanSayisi = son - ilk + 1;

    for (int i = ilk; i <= son; ++i) {
        toplam += i;
    }

    double ortalama = static_cast<double>(toplam) / elemanSayisi;

    cout << "Ortalama: " << ortalama << endl;

    return 0;
}
